includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -130,9 +131,11 @@ function get_sortable_columns() {
 		switch ( $this->type ) {
 			case 'courses':
 				$columns = array(
-					'title'           => array( 'title', false ),
-					'completions'     => array( 'completions', false ),
-					'average_percent' => array( 'average_percent', false ),
+					'title'              => array( 'title', false ),
+					'completions'        => array( 'completions', false ),
+					'average_progress'   => array( 'average_progress', false ),
```

Comment:
`average_progress`, `average_grade` and `days_to_completion` don't sort correctly, so we shouldn't add them here until we fix that later.

Reply:
Didn't quite get it? Why are they not sorting correctly, nevertheless I removed these changes.

Reply:
I don't totally understand why it doesn't work, but if you sort those columns in the UI it doesn't sort in ascending / descending order.

Reply:
@donnapep  aaaaaa I understan now !! removig everything.